### PR TITLE
Selectively build MVT

### PR DESCRIPTION
**Pull request by whirish**
New script pulls updated STVs over the past N minutes, and rebuilds mbtiles selectively for them.

### CODE REVIEW

File: data/scripts/getSTVGeoJSON.sh [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
 
-psql='psql -U postgres -t -c'
+psql="psql -d \"host=db user=${POSTGRES_USER} password=${POSTGRES_PASSWORD}\" -t -c"
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+DIR="$( cd "$( dirname "$0" )" && pwd )"
 DATA="${DIR}/../data"
 mkdir -p $DATA
 LAYER='stv'
 FCFILE="${DATA}/${LAYER}.json"
```
Comment (MiklerGM) [inline data/scripts/getSTVGeoJSON.sh]: I would propose to make some kind of lock, to eliminate possibility of the race condition as this script can be invoked via crond and through Makefile call

File: config/mvt.cron [hunk 2]
```diff
@@ -0,0 +1 @@
+0 * * * * /data/scripts/pullUpdatedSTVs.sh
```
[the remark this answers is not in the document]
Comment (whirish) [reply config/mvt.cron]: Yes it is, and sound's good

**Comment by whirish** [comment]
@MiklerGM Latest commit mounts the file in root crontab. However it appears crond is not running in the container
```
/etc/crontabs # ps -ef | grep crond
   25 root       0:00 grep crond
```
I guess you will need to add `CMD ["crond", "-f", "-d", "8"]` to the image's Dockerfile?

**Pull request merged**